### ISSUE TITLE
External dependency crypto-gost tries to inject script to background page

### ISSUE THREAD

**Issue by Sneezry** [Authenticator-Extension/Authenticator#700]
TL;DR:

crypto-gost lib tries to load `gostEngine.js` without Worker mode.

![image](https://user-images.githubusercontent.com/1621293/124921201-bb445d00-e02a-11eb-8821-4db4ed213b2e.png)

Detail:

crypto-gost needs its own engine component locates inside the module folder. But there seems a code bug in this lib, and the worker check never works.

![image](https://user-images.githubusercontent.com/1621293/124921620-29891f80-e02b-11eb-91a7-d70bff38f1ff.png)

The condition used in L6464 is always `false`, and `worker` is always `undefined` so that the lib always try to inject `gostEngine.js` with HTML tag.

This lib is hosted in a private repo, and we cannot report bugs: https://github.com/vostokplatform/crypto/issues

**Comment by Sneezry** [comment]
Hi @dmitrykatenko , do you have any idea how we can contact with `crypto-gost`'s developer for this issue? Thanks.

**Comment by dmitrykatenko** [comment]
Hi,
the same repository is located here (https://github.com/rudonick/crypto), but if you look that the last commit was in 2016, it is unlikely that it will help us with anything...

**Comment by dmitrykatenko** [comment]
I think that we can fork this repo and create new one with updates.
Do we have any deadline?

**Comment by Sneezry** [comment]
Hi @dmitrykatenko , I find another npm package based on rudonick/crypto can solve this problem. Could you help to review the PR: https://github.com/Authenticator-Extension/Authenticator/pull/707
